app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddFootwayPartSurface.kt

```diff
@@ -25,6 +25,7 @@ class AddFootwayPartSurface : OsmFilterQuestType<SurfaceAnswer>() {
             and !note:footway:surface
           )
         )
+        and access !~ private|no
```

Comment:
what about `access=no foot=yes`  taggings? Or `foot=designated`? (some RoW in England have such tagging from wha)

Reply:
`and access !~ private|no and (!foot or foot !~ private|no)` would be a better filter then, I guess?

Reply:
Watch your brackets, mister! ;-)

Reply:
I think it should be

```suggestion
        and (access !~ private|no or foot !~ private|no)
```